### ISSUE TITLE
Select field doesn't work with id = 0

### ISSUE THREAD

**Issue by Aareksio** [icebob/vue-form-generator#234]
You can't set `id: 0` in select type field values.

```JS
{
  label: 'WCF Binding',
  type: 'select',
  model: 'WCFBinding',
  values: [
    { id: 0, name: 'NetTcp' },
    { id: 1, name: 'BasicHttp' },
    { id: 2, name: 'WSHttp' }
  ],
  default: 0,
}
```

The check [here](https://github.com/icebob/vue-form-generator/blob/master/src/fields/core/fieldSelect.vue#L30) fails when `item.id = 0` and instead of `id` property alone, returns the whole object, so we end up with `[Object object]` in `value` HTML field...

It was already mentioned in #186, without reaction, so perhaps it's intended. If that's the case, please provide information on how to achieve the expected result (value = 0).

**Comment by icebob** [comment]
Please make a fiddle with the issue.

**Comment by Aareksio** [comment]
Here: https://jsfiddle.net/0mg1v81e/514/

**Comment by icebob** [comment]
Thanks. Fixed

**Comment by icebob** [comment]
Released in [v2.1.1](https://github.com/icebob/vue-form-generator/releases/tag/v2.1.1)
